### PR TITLE
Feature: Added shortcuts for Files

### PR DESCRIPTION
**Pull request by yaira2**
## Added Files

Added the default shortcuts present in Files. These shortcuts are sourced from the actions page in File Settings. However, there may be additional shortcuts that were missed.

Square44x44Logo targetsize-64

### CODE REVIEW

File: Shortcuts/apps.json [hunk 1]
```diff
@@ -991,6 +991,24 @@
...
+        "searchKeywords": [
+          "File",
+          "Manager",
+          "Tabs",
+          "Columns",
+          "Preview"
+        ],
+        "shortcutsFile": "Utilities & Tools/files.json",
+        "appDetectionInfo": {
+          "applicationIdentifier": "files.exe",
```
Comment (veler) [inline Shortcuts/apps.json:1009]: I would recommend to change this to `\\\\files\\.exe$`, which makes sure the path ends by `\files.exe`.

File: Shortcuts/Utilities & Tools/files.json [hunk 2]
```diff
@@ -0,0 +1,828 @@
...
+          ]
+        ],
+        "searchKeywords": [
+          "Fullscreen"
+        ]
+      },
+      {
+        "internalName": "Enter compact overlay\r\n",
+        "displayNames": {
+          "en-us": "Enter compact overlay\r\n"
```
Comment (veler) [inline Shortcuts/Utilities & Tools/files.json:22]: Please avoid `\r\n`. Not sure how they got here. Maybe a bug in CtrlHelpEditor. 😅

File: Shortcuts/apps.json [hunk 3]
```diff
@@ -991,6 +991,24 @@
         "appDetectionInfo": {
           "applicationIdentifier": "\\\\Todoist\\.exe$"
         }
+      },
+      {
+        "internalName": "Files",
+        "appDisplayName": "Files",
+        "logo16": "ms-appx:///Assets/Shortcuts/Icons/Square44x44Logo.targetsize-16.png",
```
Comment (veler) [inline Shortcuts/apps.json:998]: Can we please rename these files to something like `Files-16.png` and `Files-64.png` ? I will try to change CtrlHelp Editor so it does it automatically in the future.

File: Shortcuts/Utilities & Tools/files.json (hunk 2 above)
Comment (veler) [reply Shortcuts/Utilities & Tools/files.json:22]: Indeed a bug in CtrlHelp Editor. Fixing it.

File: Shortcuts/apps.json (hunk 1 above)
Comment (yaira2) [reply Shortcuts/apps.json]: Pushed 👍

File: Shortcuts/Utilities & Tools/files.json (hunk 2 above)
Comment (yaira2) [reply Shortcuts/Utilities & Tools/files.json]: Lol, I noticed this but thought it was by design. I pushed a fix.

File: Shortcuts/apps.json (hunk 3 above)
Comment (yaira2) [reply Shortcuts/apps.json]: Done 👍

**Review by veler — APPROVED** [review]

**Comment by veler** [comment]
Thank you for this contribution!

**Pull request merged**